lib/WUI/wui.cpp

```diff
@@ -384,6 +400,26 @@ class NetworkState {
                 sntp_client_step();
             }
 
+            if (events & HealthCheck) {
+                espif_tick();
+
+                // It's OK if the ESP is turned off on purpose or if it's up and running.
+                const bool esp_ok = (iface_mode(ifaces[NETDEV_ESP_ID]) == Mode::Off || ap.ssid[0] == '\0' || espif_link());
+
+                const uint32_t now = sys_now();
+                if (esp_ok) {
+                    last_esp_ok = now;
+                }
+
+                const uint32_t faulty_for = now - last_esp_ok;
```

Comment:
How about the 32bit boundary overflow? Does this computation work correctly in such a case?

Reply:
I _think_ it does, but I always find these considerations mind-boggling, so if you recheck after me, it would be nice.

Reply:
Me too :wink: , but it seems to be ok (considering only 8bits):
`0x01 - 0xfe = 3`